app/config/parameters.yml.dist

```diff
@@ -1,10 +1,10 @@
 parameters:
     database_driver:   pdo_mysql
-    database_host:     127.0.0.1
+    database_host:     db
```

Comment:
Add a comment with "You may want to change to 127.0.0.1 if you don't use docker)